### ISSUE TITLE
Can not equip an item that is stacked

### ISSUE THREAD

**Issue by Petricius** [WarEmu/WarBugs#19746]
Hello,
I can't **insert any talisman into my armor**.
It says: "_Cannot equip an item that is stacked. Please report this to the bugtracker._"
But still, I am able to put thees talis to the vault and back, So it seems "just" like armor problem.
And the talisman is not stacked, so I mean I have only one in a position (see image). 

Character name - Grombrrindal.

![issue](https://user-images.githubusercontent.com/17072092/149969669-18516c1e-4467-4232-bc46-05ddabb34cb2.png)

**Comment by lilimx** [comment]
Hey,

I can't see which talisman you mean?
could you do a picture where you want to add the talisman into the armor?

**Comment by Petricius** [comment]
Hello and thanks for swift response.
Still I can not make a screen with mouse holding the talisman, but I realized also other issues with map locations rendering etc. So I repaired files and it works now just fine.
I apologize for the wrong issue. I've been working on it since this morning.
And I didn't know what to do, but when I noticed other mistakes, I solved it like this.
Thank you so much for your time.
